bench/tx-generator/src/Cardano/Benchmarking/GeneratorTx/Genesis.hs

```diff
@@ -26,7 +26,7 @@ genesisFunds :: forall era. IsShelleyBasedEra era
   => NetworkId -> ShelleyGenesis StandardShelley -> [(AddressInEra era, Lovelace)]
 genesisFunds networkId g
  = map (castAddr *** fromShelleyLovelace)
-     $ ListMap.toList
+     $ Map.toList
```

Comment:
This removes an optimization. Do you intend to merge this PR into master?

Reply:
@Jimbo4350 it does. It's basically going back to ledger/1.0.0. 

My tentative plan is to next bump ledger to the 1.1.0 release, (and have a proper CHaP release) for it.
Then bump network to some release, though the situation there looks much less clear to me.

And then revert 9d729d8573a23bb0358f56406c488b81dc576311, at which point this is hopefully much closer to where we are _today_ with master.

If we can re-arrange the imports in a way to get ListMap from ledger/1.0.0, that would help reduce the carnage introduced in 9d729d8573a23bb0358f56406c488b81dc576311

Reply:
That being said, if we can come to an agreement that we revert to ledger/1.0.0 and the network as per 1.35.4 for now in master. And then have a second PR to try and bring in new _released_ dependencies of ledger and network, that would _certainy_ help. But it does mean we'll go back in time a bit.